eng/pipelines/runtimelab/install-jco.ps1

```diff
@@ -0,0 +1,12 @@
+$RootPath = $Args[0]
+
+$NpmExePath = $Env:NPM_EXECUTABLE
+
+Set-Location -Path $RootPath
+
+$NpmCommand = "$NpmExePath install @bytecodealliance/jco"
+Invoke-Expression $NpmCommand
```

Comment:
FWIW, this can be done via the `&` operator in PS:
```
& $NpmExePath install @bytecodealliance/jco
```

Reply:
TIL; thanks!  I'll update it.